src/ucs/datastruct/ptr_map.inl

```diff
@@ -119,47 +290,26 @@ ucs_ptr_map_put(ucs_ptr_map_t *map, void *ptr, int indirect,
  *
  * @return UCS_OK if found, UCS_ERR_NO_PROGRESS if this key is direct and
  *         therefore no action was performed, UCS_ERR_NO_ELEM if not found.
+ *
+ * @note The method moves pointer to unsafe hash table, if the pointer was
+ * found in the thread safe table.
```

Comment:
but only if the map was initialized as a safe one, right?

Reply:
yes. actually this note is not useful here (copy-paste issue).